Source/WebCore/platform/mediastream/webrtcorg/RealtimeMediaSourceCenterWebRtcOrg.cpp

```diff
@@ -90,17 +188,57 @@ void RealtimeMediaSourceCenterWebRtcOrg::validateRequestConstraints(MediaStreamC
     client->constraintsValidated(audioSources, videoSources);
 }
 
+rtc::scoped_refptr<webrtc::MediaStreamInterface> RealtimeMediaSourceCenterWebRtcOrg::createMediaStream(const std::string& audioDeviceID, 								const std::string& videoDeviceID)
+{
+  
+    ensurePeerConnectionFactory();
+    // TODO: generate unique ids using WebCore's UUID
+    const std::string streamLabel = "ToDo-Generate-Stream-UUID";
```

Comment:
Please replace "ToDo-..." strings with createCanonicalUUIDString() in this function.